### PR TITLE
Add media service + submodule (tested)

### PR DESCRIPTION
**Pull request by legendofa**

### CODE REVIEW

File: docker-compose-dev.yaml [hunk 1]
```diff
@@ -277,6 +277,26 @@ services:
       - "./.dapr/dapr-config.yaml:/config.yaml"
       - "./.dapr/components:/components"
 
+  media:
+    extends:
+      file: media/docker-compose-base.yaml
+      service: media
+    ports:
+      - 8080:8080
```
Comment (nk-coding) [inline docker-compose-dev.yaml:285]: this port is already used by the gateway
also why expose any additional ports here anyway?

Comment (legendofa) [reply docker-compose-dev.yaml:285]: Yeah, that ended up there while testing

File: docker-compose.yaml [hunk 2]
```diff
@@ -18,7 +18,6 @@ services:
     extends:
       file: frontend/docker-compose-base.yaml
       service: frontend
-    image: ghcr.io/misarch/frontend:main
```
Comment (nk-coding) [inline docker-compose.yaml:21]: ?

File: docker-compose.yaml [hunk 3]
```diff
@@ -294,6 +293,25 @@ services:
...
+    extends:
+      file: media/docker-compose-base.yaml
+      service: media
+    image: ghcr.io/misarch/media:main
+  media-minio:
+    extends:
+      file: media/docker-compose-base.yaml
+      service: media-minio
+    ports:
+      - 9000:9000
```
Comment (nk-coding) [inline docker-compose.yaml:306]: Do we need this port mapping?

**Review by nk-coding — CHANGES_REQUESTED** [review]

File: docker-compose.yaml (hunk 2 above)
Comment (legendofa) [reply docker-compose.yaml:21]: This must be a copy/paste error

File: docker-compose.yaml (hunk 3 above)
Comment (legendofa) [reply docker-compose.yaml:306]: Not necessarily if everything this is only accessed in the Docker (or whatever other internal) network

File: docker-compose.yaml (hunk 2 above)
Comment (legendofa) [reply docker-compose.yaml:21]: No idea how that ended up in there

Comment (legendofa) [reply docker-compose.yaml:21]: Need to be more careful what changes i make while debugging

**Review by nk-coding — APPROVED** [review]

**Pull request merged**